src/test/java/com/google/cloud/tools/managedcloudsdk/install/WindowsInstallScriptProviderTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.managedcloudsdk.install;
+
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class WindowsInstallScriptProviderTest {
+
+  @Test
+  public void testGetCommandLine() {
+    List<String> commandLine = new WindowsInstallScriptProvider().getScriptCommandLine();
+    Assert.assertEquals(3, commandLine.size());
+    Assert.assertEquals("cmd.exe", commandLine.get(0));
+    Assert.assertEquals("/c", commandLine.get(1));
+    Assert.assertEquals("google-cloud-sdk\\install.bat", commandLine.get(2));
```

Comment:
could use  File.pathSeparator

Reply:
I think it's OK to have `\`, as this class is not portable but strictly Windows-specific. Also, we won't replace every `/` (the Unix-counterpart) in our code with `File.pathSeparator`; we've been just using `/` almost all other cases.